packages/opentelemetry-basic-tracer/src/BasicTracer.ts

```diff
@@ -103,10 +103,17 @@ export class BasicTracer implements types.Tracer {
 
   /**
    * Returns the current Span from the current context.
+   *
+   * If there is no Span associated with the current context, null is returned.
    */
-  getCurrentSpan(): types.Span {
-    // Get the current Span from the context.
-    return this._scopeManager.active() as types.Span;
+  getCurrentSpan(): types.Span | null {
+    // Get the current Span from the context or null if none found.
+    const current = this._scopeManager.active();
+    if (current === null || current === undefined) {
```

Comment:
I'd suggest something like this?
```
if (!current) return null
return current as types.Span;
```

Reply:
Checking against specific values perform better, see https://jsperf.com/checking-for-null-or-undefined